### ISSUE TITLE
ied run PATH problem

### ISSUE THREAD

**Issue by zhuangya** [alexanderGugel/ied#11]
for the `ied` itself:

```
~> ied run test
/usr/local/lib/node_modules/ied/lib/run_cmd.js:32
    if (err) throw err
             ^

Error: spawn mocha && standard ENOENT
    at exports._errnoException (util.js:860:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)
    at Function.Module.runMain (module.js:459:11)
    at startup (node.js:136:18)
    at node.js:972:3
```

if i switch to `npm` everything is ok:

```
~> npm test

> ied@0.0.5 test /Users/nos/Documents/Projects/ied
> mocha && standard
...
...
```

it seems that something around [here](https://github.com/alexanderGugel/ied/blob/d86e29424b0a84569aea2a9ef4daea6663894cad/lib/run_cmd.js#L19) meet some problem.. i think i can have a look at it

**Comment by zhuangya** [comment]
i'm kind of in something recently

so i am not taking this bug for now

but i'll keep an eye on it :)

**Comment by zhuangya** [comment]
it's already fixed! hooray!

in d0b4b7ece594dd5a114b56bbe787d825e8711a25 i think?

**Comment by alexanderGugel** [comment]
Fixed in d0b4b7ece594dd5a114b56bbe787d825e8711a25, sorry for not updating the issue.

**Comment by zhuangya** [comment]
it's alright and thanks for your great work
